backend/src/database/entities/personnel-function-experience.entity.ts

```diff
@@ -9,6 +9,7 @@ import {
 import { FunctionEntity } from './function.entity';
 import { PersonnelEntity } from './personnel.entity';
 import { Experience } from '../../common/enums/experience.enum';
+import { ExperienceRO } from '../../personnel/ro/experience.ro';
```

Comment:
This is not included in this PR, but I think the function-> exp entities should be one to one